modules/on_gke/main.tf

```diff
@@ -124,20 +124,23 @@ data "tls_public_key" "git_sync_public_ssh_key" {
 //  Obtain Forseti Server Configuration
 //*****************************************
 
-data "google_storage_object_signed_url" "file_url" {
-  bucket      = module.server_gcs.forseti-server-storage-bucket
-  path        = "configs/forseti_conf_server.yaml"
-  content_md5 = module.server_config.forseti-server-config-md5
+data "google_storage_bucket_object" "server_config_contents" {
+  bucket = module.server_gcs.forseti-server-storage-bucket
+  name   = "configs/forseti_conf_server.yaml"
+  depends_on = [
+    module.server_config.forseti-server-config-md5
+  ]
 }
 
+data "google_client_config" "current" {}
+
 data "http" "server_config_contents" {
```

Comment:
Can you explain how this part works? Sorry I am not familiar with it. Will this cause any breaking changes?